### ISSUE TITLE
Question about the old SeleniumGridExtrasServlet

### ISSUE THREAD

**Issue by mikyone** [groupon/Selenium-Grid-Extras#153]
Hey dima, 

it's not really an issue, it's just a interrogation. I was wondering why you removed the SeleniumGridExtrasServlet from grid extra and from Selenium It was really handy to get a overview of all the grid extra in a simple. Was it too slow or too time consuming ? 
Do you have any plan to replace it with a other page ? 
Thanks for you answer.

**Comment by dimacus** [comment]
Yes it was horribly inefficient by querying all of the proxies in a big
loop. I'm working on a more nimble solution that does tiny updates on per
proxy

On Wed, Aug 19, 2015 at 5:15 AM, mikyone <notifications@github.com> wrote:

> Hey dima,
>
> it's not really an issue, it's just a interrogation. I was wondering why
> you removed the SeleniumGridExtrasServlet from grid extra and from Selenium
> It was really handy to get a overview of all the grid extra in a simple.
> Was it too slow or too time consuming ?
> Do you have any plan to replace it with a other page ?
> Thanks for you answer.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/groupon/Selenium-Grid-Extras/issues/153>.
>



-- 
-Dima Kovalenko
-------------------------

Good judgment comes from experience, and experience comes from bad judgment.
--Frederick P. Brooks

**Comment by sohammukherjee** [comment]
Hey dima,

Any update on this?? 
Because if we can extract all the details of available nodes via an api call then it will be very very helpful.
